src/write_gcm.c

```diff
@@ -2100,24 +2100,26 @@ static monitored_resource_t *wg_monitored_resource_create_for_aws(
     const wg_configbuilder_t *cb, const char *project_id);
 
 // Fetch 'resource' from the GCP metadata server.
-static char *wg_get_from_gcp_metadata_server(const char *resource);
+static char *wg_get_from_gcp_metadata_server(const char *resource,
+    _Bool ignore_failure);
```

Comment:
C doesn't have default parameters. You need to update all call sites to explicitly pass in `0`.